packages/evm/contracts/registry/GuardiansRegistry.sol

```diff
@@ -131,6 +130,8 @@ contract GuardiansRegistry is ControlledRecoverable, IGuardiansRegistry, IERC900
     // Tree to store guardians active balance by term for the drafting process
     HexSumTree.Tree internal tree;
 
+    event Staked(address indexed user, uint256 amount, uint256 total, bytes data);
+    event Unstaked(address indexed user, uint256 amount, uint256 total, bytes data);
```

Comment:
Given that we've made the decision to move away from ERC-900 now, what do you think about dropping the `data` at the end? That way we'll also be able to simplify all the related functions that leave this param unused :).